app/server/lib/configureMinIOExternalStorage.ts

```diff
@@ -1,6 +1,7 @@
 import {wrapWithKeyMappedStorage} from 'app/server/lib/ExternalStorage';
 import {appSettings} from 'app/server/lib/AppSettings';
 import {MinIOExternalStorage} from 'app/server/lib/MinIOExternalStorage';
+import {IamAwsProvider} from 'minio/dist/main/IamAwsProvider.js';
```

Comment:
Nit: please remove the `.js` extension

```suggestion
import {IamAwsProvider} from 'minio/dist/main/IamAwsProvider';
```